docs: remove misleading reference to `useFetch` (#26748)
nuxt/nuxt: docs/1.getting-started/6.data-fetching.md

</script>
```

> The `useAsyncData` composable is a great way to wrap and wait for multiple `$fetch` to be done, and then retrieve the results of each.

Reviewer: ```suggestion
The `useAsyncData` composable is a great way to wrap and wait for multiple `$fetch` requests to be done, and then retrieve the results of each.
```

> The `useAsyncData` composable is a great way to wrap and wait for multiple `$fetch` requests to be done, and then retrieve the results of each.

Reviewer: ```suggestion
The `useAsyncData` composable is a great way to wrap and wait for multiple `$fetch` requests to be completed, and then process the results.
```
